client/my-sites/stats/mobile-promo-cards/index.jsx

```diff
@@ -2,22 +2,22 @@ import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { MobilePromoCard } from '@automattic/components';
 import { translate } from 'i18n-calypso';
 import { useEffect } from 'react';
-import wordpressSeoIllustration from 'calypso/assets/images/illustrations/wordpress-seo-premium.svg';
+import blazeDropDownIllustration from 'calypso/assets/images/illustrations/blaze-drop-down.svg';
 import PromoCardBlock from 'calypso/blocks/promo-card-block';
 import DotPager from 'calypso/components/dot-pager';
 
 import './style.scss';
 
-const EVENT_YOAST_PROMO_VIEW = 'calypso_stats_wordpress_seo_premium_banner_view';
+const EVENT_BLAZE_PROMO_VIEW = 'calypso_stats_blaze_banner_view';
 const EVENT_MOBILE_PROMO_VIEW = 'calypso_stats_traffic_mobile_cta_jetpack_view';
 
-export default function MobilePromoCardWrapper( { isJetpack, isOdysseyStats, slug } ) {
-	// Yoast promo is disabled for Odyssey & self-hosted.
+export default function MobilePromoCardWrapper( { isOdysseyStats, slug } ) {
+	// Blaze promo is disabled for Odyssey.
 	// Mobile apps promos are always shown.
-	const showYoastPromo = ! isOdysseyStats && ! isJetpack;
-	const showBothPromoCards = showYoastPromo;
+	const showBlazePromo = ! isOdysseyStats;
```

Comment:
Hi @a8ck3n 👋 I noticed you just modified these files. Thought I'd ping you so you know what we're up to. :) Also, I'm honestly not sure what OdyseeyStats is and if we want our Blaze banner showing there or not. Any input here?

Reply:
Hey @DustyReagan, thanks for the heads up!

OdysseyStats is the new architecture we are putting in place so we can use the updated Calypso Stats inside wp-admin. The idea being we don't want to code two separate versions of the UI. You can read a bit more about it here: pejTkB-5e

As for these changes, have you looped in @Automattic/wpcom-marketplace so they know about their banner potentially going away? When we (Red Team) touched base with them last, we decided to go with the paged UI so that banner would still be in place.

The changes test well for me on both Atomic & self-hosted Jetpack. One possible change would be to keep the Yoast banner in play but push it to the second page of the DotPager UI. So on a self-hosted site you'd get [Blaze | Jetpack Mobile] while on an Atomic site you'd get [Blaze | Yoast | Jetpack Mobile]. Something to consider at least.

cc: @jeffgolenski @keoshi

Reply:
@DustyReagan can we keep Yoast in the banner as the next item after blaze? 

I can see that Yoast banner has more than 100 clicks daily in tracks dashboard for `calypso_stats_wordpress_seo_premium_banner_click` event.

Reply:
> @DustyReagan can we keep Yoast in the banner as the next item after blaze?
> 
> I can see that Yoast banner has more than 100 clicks daily in tracks dashboard for `calypso_stats_wordpress_seo_premium_banner_click` event.

@cpapazoglou Some context for the Blaze replacement:
- p1668541170792939/1668531558.877439-slack-C029JEQRVRT
- p58i-dut-p2

Leaving Yoast as an item the user can page through to sounds good.